README.md

```diff
@@ -129,7 +134,8 @@ ioc fetch -r custom -b
 
 ## Documentation
 
-The [API Reference (html)](https://iocage.github.io/libiocage) documenting all public interfaces of libiocage is updated with every release. The information found in the reference is compiled from Python docstrings and MyPy typings using Sphinx.
+The [API Reference (html)](https://iocage.github.io/libiocage) documenting all public interfaces of libiocage is updated with every release.
```

Comment:
i just realised: do we not own iocage.io

Reply:
Correct, will update the URL accordingly.